### PR TITLE
Maria Thomasson & Darya Lapata - Project Movies 

### PR DESCRIPTION
**Pull request by DALA746**
Link to my deployed project: https://the-movie-db-project.netlify.app/

My reflections on how this week's project turned out:
Went well, good collaboration work! 

Things I'd like to have clarified or explained in more detail:
Is there something that we can improve or maybe use another approach? If you have another solution please reach out :)

### CODE REVIEW

File: TODO.txt [hunk 1]
```diff
@@ -0,0 +1,69 @@
+TO DO LIST:
```
Comment (Rosanna86) [inline TODO.txt:1]: Nice with a to do list!

File: code/src/components/Header.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import { NavLink } from 'react-router-dom'
+
+import './header.css'
+
+export const Header = () => {
+
+    return (
+	<header className="header">
```
Comment (Rosanna86) [inline code/src/components/Header.js:9]: Nice solution with the header.

File: code/src/components/MovieDetails.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+export const MovieDetails = () => {
+
+	const { movieId } = useParams();
+	const history = useHistory(); 
+
+  	const [details, setDetails] = useState([]);
+	const [loading, setLoading] = useState(true);
+
+	const detailStyling = {
+		backgroundImage: `linear-gradient(rgba(0, 0, 0, 0) -70%, rgb(0, 0, 0) 100%), url(https://image.tmdb.org/t/p/w1280/${details.backdrop_path})`,
```
Comment (Rosanna86) [inline code/src/components/MovieDetails.js:19]: Good that you tried out styled components. But remember to do all or nothing, be consistent.

File: code/src/components/MovieDetails.js [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+		})
+	},[movieId]);
+
+
+    return (
+	<>
+	{loading && <Loading />} 
+  	<section className="summary-section">
+			<div
+					className="background-container"
```
Comment (Rosanna86) [inline code/src/components/MovieDetails.js:44]: To make it easier to read your code perhaps make the elements more inline.

File: code/src/components/MovieDetails.js [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+					>
+				<article className="info">
+					<div className="details-overall">
+						<img className="poster" src={`https://image.tmdb.org/t/p/w342/${details.poster_path}`} alt={details.title} />
+					<div className="details">
+            <h2 className="detail-title">{details.title}</h2>
+						<span className="detail-rating"> {details.vote_average}/10</span>
+            <h4>"{details.tagline}"</h4>
+            <p className="overview">{details.overview}</p>
+            <h5><span>Length:</span> {details.runtime} minutes</h5>
```
Comment (Rosanna86) [inline code/src/components/MovieDetails.js:55]: Nice detail with the length of the movie!

File: code/src/components/movieDetails.css [hunk 6]
```diff
@@ -0,0 +1,134 @@
...
+  color: white;
+  margin-left: 32px;
+}
+
+.poster {
+  margin-top: 50px;
+  margin-left: 32px;
+  border-radius: 0px;
+  background: #e0e0e0;
+  box-shadow:  20px 20px 60px #bebebe,
```
Comment (Rosanna86) [inline code/src/components/movieDetails.css:64]: Nice effect with white shadow behind the poster

File: code/src/components/movieList.css [hunk 7]
```diff
@@ -0,0 +1,113 @@
...
+  	width: 100%;
+  	height: 100%;
+}
+
+.image {
+	width: 100%;
+	display: block;
+}
+
+.overlay {
```
Comment (Rosanna86) [inline code/src/components/movieList.css:21]: Nice styling on the text overlay

**Review by Rosanna86 — COMMENTED** [review]
Really nice styling of your app. Easy code to read and nice that you added top rated movies, which was black lever requirements. 

You had a similar approach to this project that we did so we don't have any further advice. Great job! //Karolin & Rosanna

File: code/src/components/MovieDetails.js (hunk 3 above)
Comment (DALA746) [reply code/src/components/MovieDetails.js:19]: It is not a styled component, Maria found a solution to put all of the styling in an object instead :)

File: code/src/components/MovieDetails.js (hunk 4 above)
Comment (DALA746) [reply code/src/components/MovieDetails.js:44]: Don't know why it looks like this, but in VScode everything looks fine :)

**Pull request closed without merging**